docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,29 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
```

Comment:
use `ctest -jN -L unit` to run in parallel

Reply:
actually just run `ctest` to run all the tests